### ISSUE TITLE
Accuracy of reproduced baseline_imagenet_mamlresnet, baseline_imagenet_mamlconvnet, baseline_imagenet_resnet are too low.

### ISSUE THREAD

**Issue by HayeonLee** [google-research/meta-dataset#25]
After successfully converting all dataset following [this guide](https://github.com/google-research/meta-dataset#downloading-and-converting-datasets), we are trainining 3 baseline networks (baseline_imagenet_mamlresnet, baseline_imagenet_mamlconvnet, baseline_imagenet_resnet) to get pretrained chekpoints with [thie guide](https://github.com/google-research/meta-dataset/blob/master/doc/reproducing_best_results.md#baseline-and-pre-training-on-imagenet).

![image](https://user-images.githubusercontent.com/19198395/64227087-09fbaa00-cf1d-11e9-853c-1a084277be96.png)
Iteration steps are about 20K now and accuracies are under 30%. However, we think these accuracies are too low so the models do not look like to reach the reasonable accuracies even if training is finished with 50k. 

Is this training process normal? if not, could you give me some advice?

**Comment by lamblin** [comment]
This seems in line with our own training curves as well.
With baseline models on imagenet, resnet models got about 27-28% valid performance by step 20k, with a peak before 50k of 29-30%. convnet models were about 25% at 20k, 26-27% before 50k.
As reported [the main table](https://github.com/google-research/meta-dataset#models-trained-on-ilsvrc-2012-only), test performance on ILSVRC is usually quite higher than validation.
